packages/components/src/Popover/Popover.tsx

```diff
@@ -50,26 +50,26 @@ export function Popover({
     },
   );
 
-  return (
-    open && (
-      <div
-        role="dialog"
-        ref={setPopperElement}
-        style={popperStyles.popper}
-        className={classes.popover}
-        {...attributes.popper}
-      >
-        <div className={classes.dismissButton}>
-          <ButtonDismiss onClick={onRequestClose} ariaLabel="Close dialog" />
-        </div>
-        {children}
-        <div
-          ref={setArrowElement}
-          className={classes.arrow}
-          style={popperStyles.arrow}
-        />
+  return open ? (
+    <div
+      role="dialog"
+      ref={setPopperElement}
+      style={popperStyles.popper}
+      className={classes.popover}
+      {...attributes.popper}
+    >
+      <div className={classes.dismissButton}>
+        <ButtonDismiss onClick={onRequestClose} ariaLabel="Close dialog" />
       </div>
-    )
+      {children}
+      <div
+        ref={setArrowElement}
+        className={classes.arrow}
+        style={popperStyles.arrow}
+      />
+    </div>
```

Comment:
You should be able to just return `null` or `undefined`

Reply:
unfortunately can't return null cuz of the lint and undefined causes us to run into the same error as `false | JSXElement` 😭